### DISCUSSION TITLE
Papago need API Key to use? i have client ID and client secret.

### DISCUSSION THREAD

**Discussion by NninjaLLENN** [SubtitleEdit/subtitleedit#7833] [Q&A]
![image](https://github.com/SubtitleEdit/subtitleedit/assets/108386264/8ad38c85-2bbe-47d9-bde7-3bcb6ef3579d)

**Comment by niksedk** [comment]
This should be fixed in latest beta: https://github.com/SubtitleEdit/subtitleedit/releases/download/4.0.3/SubtitleEditBeta.zip
Let me know how it works

**Reply by gamania31610** [reply]
I have same question when use 4.0.3 version
![123](https://github.com/SubtitleEdit/subtitleedit/assets/24855017/93c82708-7f88-4b2e-8c2b-8c74ca76c583)

**Reply by niksedk** [reply]
Again... use the latest beta: https://github.com/SubtitleEdit/subtitleedit/releases/download/4.0.3/SubtitleEditBeta.zip
